### ISSUE TITLE
docs/README.md errors

### ISSUE THREAD

**Issue by mrvanes** [SUNET/SATOSA#66]
It seems the Docker image is not the preferred installation route anymore and this should be reflected in the docs. Also, -h is not a valid option for satosa-saml-metadata, it is --help.
Third: on Ubuntu 16.10, python 2.7 is the default python interpreter and causes some awkward python versioning entanglement. A solid, working python3 environment can be built using:

```
# apt install python3 python3-setuptools virtualenv python3-pip
# virtualenv -p /usr/bin/python3 py3env
# source py3env/bin/activate
(py3env) # pip install satosa
```

**Comment by johanlundberg** [comment]
See also #65 with regards to the Docker image.

Thank you for your report. We will update our documentation.

Also as a side note, as a Python developer I always recommend using virtualenv for development or production. Even if the underlying OS supports the needed version of Python.

**Comment by johanlundberg** [comment]
Should be fixed with PR #75.
